docs/contribute/release.md

```diff
@@ -1,2 +1,8 @@
-# Release Process
+# Release
 
+Raccoon release tags follow [SEMVER](https://semver.org/) convention.
+
+Github workflow is used to build and push the built docker image to Docker hub.
+
+A release is triggered when a github tag of format `vM.m.p` is pushed. After release job is succeeded, a docker image of
```

Comment:
1. We should also be explicit that a release have to be accepted from the committers.
2. We should provide the command to push tags (say from which branch do we release)

3. The current flow pushes the version, latest as the tags: If built from the branch, it may override the released version?  For integration/branches, shall we use commit tags instead of x.y.z & latest docker tags?

Reply:
1. Can you elaborate on this?
3. I think we should only release from master. Thus, the commit tag is not necessary.

Reply:
agree, we can release only from master.

Reply:
we can remove 2) since the committers take ownership of release.